containers/podman-desktop
# feat: allow entrypoint and cmd when starting container

### What does this PR do?

Allow to update entrypoint and cmd when starting containers

### Screenshot/screencast of this PR

![cmd](https://github.com/containers/podman-desktop/assets/695993/295d0636-ab6a-49e9-9a35-edf34956a435)

### What issues does this PR fix or reference?

Fixes #958

### How to test this PR?

1. Pull image registry.access.redhat.com/ubi8/ubi-minimal
2. When starting the image, replace `/bin/bash` by `ls`
3. See the output
